etc/html/HELP.html

```diff
@@ -176,9 +176,9 @@ <h4><a id="class_mem_access"></a>Access (public / protected / private)</h4>
 
 <h4><a id="class_mem_inh"></a>Inheritance</h4>
 You can often access members of a class's base classes, just as if they are defined
-in the derived class. A <a href="http://root.cern.ch/root/html/TH1.html">histogram</a>,
+in the derived class. A <a href="http://root.cern/root/html/TH1.html">histogram</a>,
 for example, has a name, and you can access it using GetName() as defined in its base
-class <a href="http://root.cern.ch/root/html/TNamed.html">TNamed</a>. If you want to see
+class <a href="http://root.cern/root/html/TNamed.html">TNamed</a>. If you want to see
```

Comment:
Correct link : https://root.cern/doc/master/classTNamed.html

Reply:
The other one also works